commands/blacklist.js

```diff
@@ -27,6 +27,14 @@ exports.run = async (client, message, args) => {
     await client.db.updateUser(user.data)
 
     message.channel.send(`${greentick} ${client.utils.escapeMarkdown(user.tag)} (\`${user.id}\`) has been blacklisted`)
+    
+    //leave his server
+    try {
+      client.guilds.filter(c=>c.owner.id === user.id).leave()
```

Comment:
This will likely say `<Array>.leave() is not a function`. Put the leaves in a `forEach`.

Reply:
ooo okkk